tests/unit/test_log.py

```
@@ -67,20 +67,102 @@
+
+        with self.subTest("older log file deleted"):
+            self.assertFalse(
+                os.path.exists(
+                    Path(TEST_DATA_DIR).joinpath(
+                        f"s3_upload.log.{six_days_ago}"
+                    )
+                )
+            )
+
+    @patch("s3_upload.utils.log.os.remove")
+    def test_errors_on_deleting_caught_and_logged_but_not_raised(
```

Ruff (0.8.0)

Missing return type annotation for public function `test_errors_on_deleting_caught_and_logged_but_not_raised`

Add return type annotation: `None`

(ANN201)